backend/infrahub/core/protocols.py

```diff
@@ -480,6 +480,7 @@ class CoreProposedChange(CoreTaskTarget):
     source_branch: String
     destination_branch: String
     state: Enum
+    is_draft: Boolean
```

Comment:
I've been going back on forth with this. Wondering if we should have a boolean attribute, if `draft` should just be a new entry in the "state" enum?

The benefit of having an "is_draft" is that you can still query for all open proposed changes in one query instead of having to specify multiple states.

Reply:
We rediscussed it and as a PC can be both draft AND open/merged/closed/... , having a separated boolean looks cleaner. It's also how it's done in Github